CONTRIBUTING.md

```diff
@@ -104,11 +101,16 @@ black quantus/INSERT_YOUR_FILE_NAME.py
 ```bash
 flake8 quantus/INSERT_YOUR_FILE_NAME.py
 ```
-- Create `pytests` for new functionality (if needed) and add under `tests/` folder
-- If the `pytests` include a new category of `@pytest.mark` then add that category with description to `pytest.ini`
-- Make sure all unit tests are passed and the coverage level is maintained (we aim at ~100% code coverage for Quantus):
+- Create tests for new functionality and add under `tests/` folder.
+- If tests include a new category of `@pytest.mark` then add that category with description to `pytest.ini`
+- Make sure all unit tests pass for all supported python version by running:
+```shell
+python -m tox run
+```
+- Generally, every change should be covered with respective test-case, we aim at ~100% code coverage for Quantus.
+You can verify it by running:
 ```bash
-pytest tests -v --cov-report term --cov-report html:htmlcov --cov-report xml --cov=quantus
+python -m tox run coverage
```

Comment:
Ok good, clear here. Please add this to above commands as well so we guide the user to track coverage while development.

Reply:
done